src/uwtools/cli.py

```diff
@@ -928,6 +942,13 @@ def _check_template_render_vals_args(args: Args) -> Args:
     return args
 
 
+def _check_update(args: Args) -> Args:
+    if args.get(STR.updatefile) is not None:
+        if args.get(STR.updatefmt) is None:
+            args[STR.updatefmt] = get_file_format(args[STR.updatefile])
+    return args
```

Comment:
It's not the case that at least one of `--input-file` and `--update-file` be provided: If neither are provided, then `uw` will read the input from `stdin` and will not perform an update (it will just process the input file with internal information or potentially environment variables if the new-ish `env` filter is used). If `--update-file` is provided, `uw` will perform an update via the specified file; if `--update-format` is provided and `--update-format` is not, `uw` will read the update info from `stdin` **unless** `--input-file` is also not provided, which is then an `error`. This is documented [here](https://github.com/ufs-community/uwtools/pull/493/files#diff-a4378ed914a7ad3423e757c932d624dff449acbaff2574aaed9ec48e57c29c93R359-R363).